### ISSUE TITLE
THAT BUG THAT LET'S EVERY TYPE OF CONSTRUCT HEAL THEMSELVES NO MATTER WHAT

### ISSUE THREAD

**Issue by Super3222** [yogstation13/yogstation#1471]
I'm trying to contain myself here, bear with me.

First of all - Merry Christmas. It's the 25th and it's 2:46 AM where I am.

Anyway, getting back to it.
This "PR" - https://github.com/yogstation13/yogstation/pull/1212/files - was meant to let artificers heal themselves again. This ultimately backfired, and now every construct can heal themselves by clicking on their body. This is JUGGERNAUGHTS included, and the players KNOW OF THIS BUG, and are USING IT TO WIN BLOODCULT.

tl;dr don't copypaste code and listen to super (https://github.com/yogstation13/yogstation/issues/1248) when he tells you to let simple animals take damage (or do heal themselves via artificer) like all mobs do

**Comment by ShadowDeath6** [comment]
That's an easy pathing fix and you know it, I'll get on that asap, which is probably tomorrow night. If you can't wait that long, just add /builder before attackingself and that'll fix it.

**Comment by Super3222** [comment]
Take your time with it. I'm just putting it out there for coders alike to know
